ui/src/ReusableComponents/ProductFilter.tsx

```diff
@@ -121,19 +122,12 @@ function ProductFilter({
         tagFilterOptions: AllGroupedTagFilterOptions
     ): Array<FilterOption> {
         return tagFilterOptions.options.map(
-            option => {
-                if (selectedOptions && selectedOptions.includes(option)) {
-                    return {
-                        ...option,
-                        selected: true,
-                    };
-                } else {
-                    return {
-                        ...option,
-                        selected: false,
-                    };
+            option => (
+                {
+                    ...option,
+                    selected: selectedOptions.includes(option),
```

Comment:
I'm seeing this break:
<img width="829" alt="Screen Shot 2020-11-16 at 1 48 16 PM" src="https://user-images.githubusercontent.com/74197928/99294714-7016d900-2812-11eb-95ef-2369fd118520.png">